.github/workflows/make-all.yml

```diff
@@ -36,11 +36,14 @@ jobs:
       - uses: actions/setup-go@v3
         with:
           go-version: ${{ env.GOLANG_VERSION }}
-  
+
       - name: Install k6
         run: |
           go install go.k6.io/xk6/cmd/xk6@v${{ env.XK6_VERSION }}
-          xk6 build v${{ env.K6_VERSION }} --with github.com/grafana/xk6-sql@v0.4.1 && sudo cp k6 /usr/bin
+          xk6 build v${{ env.K6_VERSION }} \
```

Comment:
❓ @donch1989 do the `make-*` workflows need this step? I don't see where they are referenced and the workflow itself doesn't need k6